dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/compilation/VisitorPower.java

```diff
@@ -30,24 +30,24 @@
 import static com.dat3m.dartagnan.expression.op.COpBin.NEQ;
 import static com.dat3m.dartagnan.program.event.EventFactory.*;
 import static com.dat3m.dartagnan.program.event.Tag.STRONG;
-import static com.dat3m.dartagnan.program.processing.compilation.SyncScheme.LEADING;
+import static com.dat3m.dartagnan.program.processing.compilation.VisitorPower.PowerScheme.LEADING_SYNC;
 
-class VisitorPower extends VisitorBase implements EventVisitor<List<Event>> {
+public class VisitorPower extends VisitorBase implements EventVisitor<List<Event>> {
 
 	// The compilation schemes below follows the paper
 	// "Clarifying and Compiling C/C++ Concurrency: from C++11 to POWER"
 	// The paper does not defines the mappings for RMW but we derive them
 	// using the same pattern as for Load/Store	
-	private final SyncScheme scheme;
+	private final PowerScheme cToPowerScheme;
 	// If the source WMM does not allow OOTA behaviors (e.g. RC11)
 	// we need to strength the compilation following the papers
 	// "Repairing Sequential Consistency in C/C++11"
 	// "Outlawing Ghosts: Avoiding Out-of-Thin-Air Results"
-	private final boolean nooota; 
+	private final boolean useRC11Scheme; 
```

Comment:
I think the comment can get updated. LKMM, for example, seems to successfully use dependencies on the language level to avoid OOTA.
What is true is that *non-dependency* tracking models may need to use the strong version of `noOOTA`.
So maybe say that `RC11` is not dependency-tracking and hence uses a strong definition of `noOOTA`. This requires stronger compilation schemes to hardware.